vignettes/vignette_best_practices.Rmd

```diff
@@ -0,0 +1,198 @@
+---
+title: "Best practices when writing vignette in admiral"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Best practices when writing vignette in admiral}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+
+```{r setup0, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+# Introduction
+
+This guidance ensures consistency across the vignettes in the admiral package in terms of content, structure and code execution. 
+As a general rule, the vignette workflow defined in [r-pkgs.org](https://r-pkgs.org/vignettes.html) is followed.
+
+
+# Metadata
+
+Each vignette in admiral should start with the following metadata.
+
+```
+---
+title: "<Title>"
+output:  rmarkdown::html_vignette:
+vignette: >
+  %\VignetteIndexEntry{<Title>}
+  %\VignetteEngine{knitr::rmarkdown}
+---
+```
+
+The `<Title>` of the vignette should be meaningful. 
+
+# Markdown
+
+## Default Options
+
+    `r ''````{r setup, include=FALSE}
+    knitr::opts_chunk$set(
+      collapse = TRUE,
+      comment = "#>"
+      )
+    ```
+
+
+## Format Sections
+
+### Table of Contents
+
+Headings must be title case and start from Heading 1:
+
+```
+# Heading 1
+This is the description of my first section.
+
+## Heading 2.1
+This is the description of my first sub-section within my first section.
+
+## Heading 2.2
+This is the description of my second sub-section within my first section.
+
+```
+
+The first section gives a brief introduction of the vignette. The last sub-section 
+of the introduction should describe the packages required to run the admiral functions. The admiral package should be described first.
+
+```
+# Introduction
```

Comment:
I don't think our current vignettes follow this.  For example, our introduction has `##`

Reply:
This will be part of the update to be made to all vignettes (#572 ). Some vignettes starts the heading at 1, some starts at 2.